### ISSUE TITLE
Dependencies are not downloading

### ISSUE THREAD

**Issue by Pikman** [ebenbruyns/junkstore#33]
I installed it, but when I press "Install dependencies" log window stays white forever and nothing happens. If I disable wifi - I see errors, so the log window works. Reinstallation from decky did not helped. How can I solve that?

**Comment by McBruno712** [comment]
Can confirm it works from Konsole but not from the GUI.

**Comment by ebenbruyns** [comment]
Do you have the same behaviour when turning the wifi off? The issue might not be what I thought it was if that's the case.

**Comment by McBruno712** [comment]
Yep, if I disable WiFi I can see the log on the box, but not if I'm connected, it's like the script never runs.

**Comment by ebenbruyns** [comment]
It obviously run, because you get the error outputs, but it seems something is "blocking" it for some reason. It's probably going to be something like an environment variable that's not set in all cases. This is going to be a tricky one to find I can tell.

**Comment by remoteman213** [comment]
Same issue for me, blank log window in GUI

**Comment by ebenbruyns** [comment]
Can you provide more details beyond that to help me track it down?

Regards,

Eben Bruyns

**Comment by remoteman213** [comment]
OK, so I did some testing and it looks like it does run, but doesn't output to the text box till the script is complete. Took about an hour with my internet speed.

**Comment by MrEAlderson** [comment]
I had to wait like two minutes for anything to appear on a fresh system

**Comment by mrsjunkrunner** [comment]
[Dependecies Issue noted on Decky PR v1.1.9](https://github.com/SteamDeckHomebrew/decky-plugin-database/pull/782#issuecomment-2727010427)

**Comment by mrsjunkrunner** [comment]
This issue appears to be resolved. There haven't been any users who are experiencing this issue any more. We can reopen if this issue comes up again in future.
